be/src/exprs/vectorized/binary_function.h

```diff
@@ -471,14 +471,14 @@ class LogicPredicateBinaryFunction {
             ln = p->null_column();
         } else if (v1->is_constant()) {
             ld = ColumnHelper::as_raw_column<ConstColumn>(v1)->data_column();
-            ln = ColumnHelper::s_all_not_null_column;
+            ln = NullColumn::create(v1->size(), 0);
```

Comment:
This change may affect performance and increase unnecessary memory allocation

Reply:
Revoke to use ColumnHelper::s_all_not_null_column, and use chunk_size_all_not_null_column = 16384 to instantiate ColumnHelper::s_all_not_null_column.